client/src/components/Contact.js

```diff
@@ -1,14 +1,40 @@
-import React, { Component, Fragment } from 'react';
+import React, { Fragment } from 'react';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import * as S from './Contact.style';
 import CHeader from './CHeader';
+import CFooter from './CFooter';
 
-class Contact extends Component {
-  render() {
-    return (
-      <Fragment>
-        <CHeader menu />
-      </Fragment>
-    );
-  }
-}
+const Contact = () => {
+  return (
+    <Fragment>
+      <CHeader menu />
+      <S.Main>
+        <S.H1>Contact</S.H1>
+        <S.ContactWrapper>
```

Comment:
instead of using ```div```, this might be better as ```<address>```
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address

That means this tag will be something like ```S.Address```